_posts/2019-04-17-sod.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "Apache Flink's Application to Season of Docs"
+date:   2019-4-17 12:00:00
+categories: news
+authors:
+- konstantin:
+  name: "Konstantin Knauf"
+  twitter: "snntrable"
+---
+
+The Apache Flink community is happy to announce its application to the first edition of [Season of Docs](https://developers.google.com/season-of-docs/) by Google. The program is bringing together Open Source projects and technical writers to raise awareness for and improve documentation of Open Source projects. We would like to take this chance to work with one or two technical writers for three months to improve and restructure parts of [our documentation](https://ci.apache.org/projects/flink/flink-docs-stable/) (details below).
+
+The community has discussed this opportunity on the [dev mailinglist](https://lists.apache.org/thread.html/3c789b6187da23ad158df59bbc598543b652e3cfc1010a14e294e16a@%3Cdev.flink.apache.org%3E) and agreed on three project ideas to submit to the program. We have a great team of mentors (Stephan, Fabian, David, Jark & Konstantin) lined up and are very much looking forward to the first proposals by potential technical writers (given we are admitted to the program ;)). In case of questions feel free to reach out to the community via [dev@flink.apache.org](../../../../community.html#mailing-lists).
+
+## Project Ideas List
+
+### Project 1: Improve Documentation of Stream Processing Concepts
+
+**Description:** Stream processing is the processing of data in motion―in other words, computing on data directly as it is produced or received. Apache Flink has pioneered the field of distributed, stateful stream processing over the last several years. As the community has pushed the boundaries of stream processing, we have introduced new concepts that users need to become familiar with to develop and operate Apache Flink applications efficiently.
+The Apache Flink documentation \[1\] already contains a “concepts” section, but it is a ) incomplete and b) lacks an overall structure & reading flow. In addition, “concepts”-content is also spread over the development \[2\] & operations \[3\] documentation without references to the “concepts” section. An example of  this can be found in \[4\] and \[5\].
```

Comment:
```suggestion
The Apache Flink documentation \[1\] already contains a “concepts” section, but it is a) incomplete and b) lacks an overall structure & reading flow. In addition, “concepts”-content is also spread over the development \[2\] & operations \[3\] documentation without references to the “concepts” section. An example of this can be found in \[4\] and \[5\].
```